gdx/src/com/badlogic/gdx/scenes/scene2d/ui/List.java

```diff
@@ -158,6 +161,8 @@ public void draw (Batch batch, float parentAlpha) {
 					selectedDrawable.draw(batch, x, y + itemY - itemHeight, width, itemHeight);
 					font.setColor(fontColorSelected.r, fontColorSelected.g, fontColorSelected.b, fontColorSelected.a * parentAlpha);
 				}
+				String string = item.toString();
+				layout.setText(font, string, 0, string.length(), font.getColor(), width, listAlignment, false, "...");

```

Comment:
Why is this layout never used? Also `toString(item)` should be used, not `item.toString`.